### ISSUE TITLE
How can I get accuracy

### ISSUE THREAD

**Issue by Julia2505** [ageitgey/face_recognition#1176]
* face_recognition version: 1.3.0
* Python version: 3.6
* Operating System: Google Colaboratory

### Description

Hello! I'm trying to get model recognition accuracy in percentage. I have trained .clf model on LFW dataset using your open-source code: https://github.com/ageitgey/face_recognition/blob/master/examples/face_recognition_knn.py

### What I Did
I was trying to use encoders to get arrays of images names, but it was unsuccessfull... 

```
path="lfw"
from sklearn.metrics import accuracy_score
from sklearn.preprocessing import LabelEncoder
from sklearn.preprocessing import Normalizer

in_encoder = LabelEncoder()
out_encoder = LabelEncoder()

for folder in os.listdir(path):
    i = 0
    in_encoder.fit(name) #added
    out_encoder.fit(predictions) #added 
    inn = in_encoder.transform(name) #added 
    outt = out_encoder.transform(predictions) #added
    print (inn) #added
    for image_file in  os.listdir(path + "/" + folder):
        full_file_path = os.path.join(path + "/" + folder + "/" + image_file)

        print("Looking for faces in {}".format(image_file))

    for name, (top, right, bottom, left) in predictions:
            print("- Found {} at ({}, {})".format(name, left, top))

        # Display results overlaid on an image
        show_prediction_labels_on_image(os.path.join(path + "/" + folder + "/" + image_file), predictions)
```
Output:
```
bad input shape ()
```
Please, help me to solve this problem or tell other way how to get model percentage accuracy.
